### PR TITLE
sla resource

### PR DESCRIPTION
**Pull request by Abhishekism9450**
Resource for NDB SLA added .

### CODE REVIEW

File: nutanix/resource_nutanix_ndb_sla.go [hunk 1]
```diff
@@ -0,0 +1,276 @@
...
+			},
+			"date_created": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"date_modified": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"yearly_retention": {
```
Comment (bhati-pradeep) [inline nutanix/resource_nutanix_ndb_sla.go]: If yearly retention is compulsory

File: nutanix/resource_nutanix_ndb_sla_test.go [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+					resource.TestCheckResourceAttr(resourceNameSLA, "monthly_retention", "1"),
+					resource.TestCheckResourceAttr(resourceNameSLA, "quarterly_retention", "1"),
+				),
+			},
+			{
+				Config: testAccEraSLAConfigUpdated(updatedName, updatedDesc),
+				Check: resource.ComposeTestCheckFunc(
+					resource.TestCheckResourceAttr(resourceNameSLA, "name", updatedName),
+					resource.TestCheckResourceAttr(resourceNameSLA, "description", updatedDesc),
+					resource.TestCheckResourceAttr(resourceNameSLA, "continuous_retention", "30"),
```
Comment (bhati-pradeep) [inline nutanix/resource_nutanix_ndb_sla_test.go]: Can we check continous retention update here ?

**Review by bhati-pradeep — COMMENTED** [review]
Minor changes. LGTM

File: nutanix/resource_nutanix_ndb_sla.go (hunk 1 above)
Comment (Abhishekism9450) [reply nutanix/resource_nutanix_ndb_sla.go]: kept as optional and computed

**Review by siddharth-kulshrestha — APPROVED** [review]

**Pull request merged**